src/forms/SelectionList/SelectionList.jsx

```diff
@@ -0,0 +1,109 @@
+import { useState, useEffect, useMemo } from 'preact/hooks';
+import PropTypes from 'prop-types';
+
+import Heading from '../../typography/Heading/Heading';
+import CSS from './styles.css';
+
+const SelectionList = ({ className, dataTestId, value, title, options, id, onChange }) => {
+	const [currentValue, setCurrentValue] = useState(null);
+	const delayCallback = 250;
+
+	useEffect(() => {
+		if (value) {
+			setCurrentValue(value);
+			return;
+		}
+		const defaultCheckedOptionIndex = options.findIndex(({ selected }) => selected);
+		setCurrentValue(defaultCheckedOptionIndex !== -1 ? options[defaultCheckedOptionIndex].value : null);
+	}, [value, options]);
+
+	const handleOnClick = ({ selected, label, value: optionValue, children, onClick }, index) => {
+		if (onClick) {
+			setTimeout(() => onClick(optionValue, index), delayCallback);
```

Comment:
(nit) please enclose this arrow-function body within curly-braces.

Same comment for line: 25, below.